src/test/rpc/AccountTx_test.cpp

```diff
@@ -164,15 +164,299 @@ class AccountTX_test : public beast::unit_test::suite
+            {  5, jss::PaymentChannelClaim,  {},                                                         {jss::PayChannel},                {jss::AccountRoot, jss::AccountRoot,   jss::DirectoryNode}},
+            {  6, jss::PaymentChannelFund,   {},                                                         {},                               {jss::AccountRoot, jss::PayChannel   }},
+            {  7, jss::PaymentChannelCreate, {jss::PayChannel},                                          {},                               {jss::AccountRoot, jss::AccountRoot,   jss::DirectoryNode}},
+            {  8, jss::EscrowCancel,         {},                                                         {jss::Escrow},                    {jss::AccountRoot, jss::DirectoryNode}},
+            {  9, jss::EscrowFinish,         {},                                                         {jss::Escrow},                    {jss::AccountRoot, jss::DirectoryNode}},
+            { 10, jss::EscrowCreate,         {jss::Escrow},                                              {},                               {jss::AccountRoot, jss::DirectoryNode}},
+            { 11, jss::EscrowCreate,         {jss::Escrow},                                              {},                               {jss::AccountRoot, jss::DirectoryNode}},
+            { 12, jss::SignerListSet,        {jss::SignerList},                                          {},                               {jss::AccountRoot, jss::DirectoryNode}},
+            { 13, jss::OfferCancel,          {},                                                         {jss::Offer, jss::DirectoryNode}, {jss::AccountRoot, jss::DirectoryNode}},
+            { 14, jss::OfferCreate,          {jss::Offer, jss::DirectoryNode},                           {},                               {jss::AccountRoot, jss::DirectoryNode}},
+            { 15, jss::TrustSet,             {jss::RippleState, jss::DirectoryNode, jss::DirectoryNode}, {},                               {jss::AccountRoot, jss::AccountRoot}},
+            { 16, jss::SetRegularKey,        {},                                                         {},                               {jss::AccountRoot}},
+            { 17, jss::Payment,              {},                                                         {},                               {jss::AccountRoot, jss::AccountRoot}},
+            { 18, jss::AccountSet,           {},                                                         {},                               {jss::AccountRoot}},
+            { 19, jss::AccountSet,           {},                                                         {},                               {jss::AccountRoot}},
+            { 20, jss::Payment,              {jss::AccountRoot},                                         {},                               {jss::AccountRoot}},
+        };
+
+        BEAST_EXPECT (std::extent<decltype (sanity)>::value ==
```

Comment:
L340 sort of overlaps with this check...I think I like this one better since it verifies against the sanity data

Reply:
Good point.  Would you prefer that I remove the check on line 340?  Or is it okay to leave it?

Reply:
I'm weakly in favor or removing L340, but I'm fine either way.